src/utils/Api/index.js

```diff
@@ -64,21 +64,39 @@ class API {
     `;
   }
 
+  static generateFreelancerInfoQuery(id) {
+    return `
+      query {
+          allFreelancers (condition: {flId:"16c3adff-6e92-4d80-8a63-3e9536197d47"}) {
+            nodes {
+              flFirstName
+              flLastName
+              flIsNativeSpeaker
+              flLocation
+              flTimezone
+              flPrimaryLanguage
+              flEmploymentStatus
+            }
+          }
+        }
```

Comment:
This query has two issues:

- [ ] it's a statically coded UUID which means I can't test because UUIDs are different on each instances. replace with ``${id}``
- [ ] the query returns a _400 bad request_ response. 

The correct query is something like this: 

````
{
  allFreelancers(condition: {
    flId: "0ba6e94f-1956-4a94-b079-bc29fde23374"
  }) {
    edges {
      node {
        flId
        flLastName
        flFirstName
        flLocation
        flTimezone
        flIsNativeSpeaker
        flPrimaryLanguage
        flEmploymentStatus
      }
    }
  }
}
````

the response you get back is something like:

````
{ data { allFreelancers:  { edges:  [ { node: { ... } , ... ] } } }
````

Note the array called edges and then each freelancer is a node inside the array.